python/cudf/cudf/utils/ioutils.py

```diff
@@ -199,6 +199,16 @@
     in parallel (using a python thread pool). Default allocation is
     {bytes_per_thread} bytes.
     This parameter is functional only when `use_python_file_object=False`.
+skiprows : int, default None
+    If not None, the number of rows to skip from the start of the file.
+
+    .. note::
+       This option is not supported when the low-memory mode is on.
```

Comment:
*nit (non-blocking)*: Not your fault, but there is no description anywhere (other than in the not very discoverable options documentation) of what low-memory mode means. So there's no way for someone to easily know what this means.